### PR TITLE
[TGI Fridays US] Fix spider

### PR DESCRIPTION
**Pull request by Cj-Malone**

### CODE REVIEW

**Comment by davidhicks** [comment]
I think you've accidentally written the replacement YextSpider that #7852 also requires :) It appears Yext are in the process of changing their customers to a new API.

**Comment by davidhicks** [comment]
> I think you've accidentally written the replacement YextSpider that #7852 also requires :) It appears Yext are in the process of changing their customers to a new API.

Hmm actually maybe not, #7852 looks different again.

**Comment by Cj-Malone** [comment]
I specifically wrote this so it could be easily split out into shared code. Do you think Yext is dropping the `entities` endpoint for all customers, I assumed only a few had it turned off.

**Comment by davidhicks** [comment]
Perhaps not a complete replacement? As this spider shows, the brand has disabled the `entities` endpoint and are instead using a different `answers/vertical` endpoint. I've seen hints at this `answers` endpoint elsewhere when writing spiders, but perhaps we've just been noticing it, then trying the `entities` endpoint instead, and it just happens to work as most brands enable the `entities` endpoint regardless of if they use the `answers` endpoint.

If we see more uses of `answers/vertical` only (no `entities` endpoint supported) I guess we could bother to create a new `YextAnswersSpider` store finder, but seemingly no benefit just yet.

File: locations/storefinders/yext_answers.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    api_version: str = "20220511"
+    page_limit: int = 50
+
+    def make_request(self, offset: int) -> JsonRequest:
+        return JsonRequest(
+            url="{}?{}".format(
+                self.endpoint,
+                urlencode(
+                    {
+                        "experienceKey": "tgi-fridays-search",
```
Comment (davidhicks) [inline locations/storefinders/yext_answers.py:28]: `experienceKey` should be another API parameter?

**Review by davidhicks — CHANGES_REQUESTED** [review]

File: locations/storefinders/yext_answers.py (hunk 1 above)
Comment (Cj-Malone) [reply locations/storefinders/yext_answers.py]: Done, thanks.

**Review by davidhicks — APPROVED** [review]

**Pull request merged**